docs/components/optimize/userguide/process-analysis/outlier-analysis.md

```diff
@@ -0,0 +1,25 @@
+---
+id: outlier-analysis
+title: Outlier analysis
+description: Outlier analysis allows you to easily identify process instances where certain flow node instances took significantly longer than others.
+---
+
+# Overview
+
+Outlier analysis allows you to easily identify process instances where certain flow node instances took significantly longer than others and subsequently slow down your process. Let’s have a look at the feature based on a step-by-step example.
```

Comment:
I expect "flow node instances" to be technically correct but too obtuse for most users. 
I would go with "Outlier analysis allows you to easily identify process instances that took significantly longer than others to complete a flow node."

Reply:
Agreed on this clarity. Updated in latest commit.